btr-api/tests/unit/resources/test_requests.py

```diff
@@ -87,24 +90,27 @@ def test_get_fail_no_auth(app, client, session, jwt, requests_mock, sample_user,
 
 @pytest.mark.parametrize(
     'test_name, user_type',
-    [('test_post', UserType.USER_COMPETENT_AUTHORITY)],
+    [('test_post', UserType.STAFF_ROLE)],
 )
 def test_post(app, client, session, jwt, requests_mock, sample_user, test_name, user_type):
     """Get the plot submissions.
     A parameterized set of tests that runs defined scenarios.
     """
     with nested_session(session):
         clear_db(session)
-        # Setup
+        session.add(sample_user)
+        session.commit()
+
         requests_mock.get(
             f"{app.config.get('AUTH_SVC_URL')}/orgs/1/products?include_hidden=true",
-            json=[{'code': 'CA_SEARCH', 'subscriptionStatus': 'ACTIVE'}])
+            json=[{'code': 'STAFF', 'subscriptionStatus': 'ACTIVE'}])
```

Comment:
Staff isn't a valid product - the product mock is only required for setting up a CA user so you don't need this mock anymore.

Also, since 'user_type' is not being used at all you can change that to be 'roles' and then use it in the 'create_header' below which would allow you to also test other users types here